src/env.ts

```diff
@@ -19,6 +20,24 @@ export const env = {
   FUSE_MODULES: path.join(FUSE_ROOT, 'modules'),
 };
 
+export function getPackageManagerData() {
+  function Data(name: string, installCmd: string, installDevCmd: string) {
+    this.name = name;
+    this.installCmd = installCmd;
+    this.installDevCmd = installDevCmd;
+  }
+
+  if (fs.existsSync(path.join(FUSE_ROOT, './.yarnrc'))
+      || fs.existsSync(path.join(FUSE_ROOT, './yarn.lock'))) {
+    return new Data('yarn', 'yarn add', 'yarn add --dev');
+  } else if (fs.existsSync(path.join(FUSE_ROOT, './pnpm-lock.yaml'))) {
+    return new Data('pnpm', 'pnpm add', 'pnpm add --save-dev');
+  } else {
+    // package-lock.json
+    return new Data('npm', 'npm install', 'npm install --save-dev');;
```

Comment:
can't you use just return `{name : "npm", ...}`
Overall it looks good, but I don't like this class here ;-)

Reply:
fixed